src/main/java/seedu/homerce/logic/commands/service/ClearServiceCommand.java

```diff
@@ -18,9 +25,25 @@
 
 
     @Override
-    public CommandResult execute(Model model, HistoryManager historyManager) {
+    public CommandResult execute(Model model, HistoryManager historyManager) throws CommandException {
         requireNonNull(model);
+
+        model.updateFilteredServiceList(PREDICATE_SHOW_ALL_SERVICES);
+        model.updateFilteredAppointmentList(PREDICATE_SHOW_ALL_APPOINTMENTS);
```

Comment:
When we call update, the UI will change if the user is on the appointments tab. Not sure if this is an acceptable outcome.

Reply:
Thanks for pointing that out! I have changed the implementation to fix this.